netket/machine/abstract_machine.py

```diff
@@ -65,6 +65,34 @@ def der_log(self, x, out=None):
             """
         return NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
Does this need to be abstract?

Reply:
No probably not, the method can be the same for every machine